tests/unit/commands/__init__.py

```diff
@@ -73,6 +73,12 @@ def setUp(self):
         self.runner = CliRunner()
 
     def run_command(self, args, **kwargs):
+        # For click testing, runner will overwrite the descriptors for stdio -
+        # ensure TTY always appears connected.
+        patcher = mock.patch("snapcraft.cli.echo.is_tty_connected", return_value=True)
```

Comment:
I have been lately preferring the more linguistic friendly use of fixtures,
```python
self.useFixture(fixtures.MockPatchObject("snapcraft.cli.echo.is_tty_connected", return_value=True))
```
If you really want to make the fixture/mock available (although I see it not being used anywhere), you can do:
```python
self.fake_is_tty = fixtures.MockPatchObject("snapcraft.cli.echo.is_tty_connected", return_value=True)
self.useFixture(self.fake_is_tty)
```
And access the mock object using with `.mock`

Reply:
I like that approach - updated to use the fixture.  Thanks!